src/db/models/User.ts

```diff
@@ -206,6 +207,12 @@ UserModel.init(
             allowNull: false,
             defaultValue: {},
         },
+        locale: {
+            type: DataTypes.STRING(),
```

Comment:
should it be an enum of what locales we're supporting ? 
should all locales supported be in a separate table and the locale of the users be a foreign key to this new table ?

Reply:
I made the change from STRING to ENUM, I think that having a table for this is a bit overkill considering the data it would contain. But if you had a plan in mind for the future let me know!